Source/WebCore/inspector/InspectorFrontendClientLocal.cpp

```diff
@@ -80,7 +80,7 @@ static const float minimumAttachedInspectedWidth = 320.0f;
 class InspectorBackendDispatchTask : public RefCounted<InspectorBackendDispatchTask> {
     WTF_MAKE_TZONE_ALLOCATED(InspectorBackendDispatchTask);
 public:
-    static Ref<InspectorBackendDispatchTask> create(InspectorController* inspectedPageController, InspectorFrontendClientLocal::DispatchBackendTarget dispatchTarget)
+    static Ref<InspectorBackendDispatchTask> create(PageInspectorController* inspectedPageController, InspectorFrontendClientLocal::DispatchBackendTarget dispatchTarget)
```

Comment:
I think you should make a drive-by fix to rename all `inspectedPageController` to `pageInspectorController` as well (or just `inspectorController`, if the scope is short). I don't think the slight reordering in the name adds value.

```suggestion
    static Ref<InspectorBackendDispatchTask> create(PageInspectorController* pageInspectorController, InspectorFrontendClientLocal::DispatchBackendTarget dispatchTarget)
```

Reply:
I'll need to revisit these files shortly to adopt CheckedRef for InspectorController, so let's keep this a simple rename.